spec/koans.js

```diff
@@ -423,21 +434,25 @@ describe("the JavaScript language", function() {
       var obj = createObject();
       obj.addPoint();
 
-      //expect(obj.score()).toEqual();
-      //expect(typeof(obj.points)).toEqual();
+      expect(obj.score()).toEqual(1);
+      expect(typeof(obj.points)).toEqual('undefined');
     });
 
     it("may create objects also with the module pattern", function() {
       function createObject(initialScore) {
-        // write the code here
+        this.points = initialScore
+        this.color = arguments[1]
+        this.incrementScoreIn = function(){
+          this.points += initialScore;
+        }
       }
 
-      /*
+      
       var obj = createObject(5, 'red');
       obj.incrementScoreIn(5);
       expect(obj.color).toEqual('red');
       expect(obj.points()).toEqual(10);
-      */
+      
     });
 
```

Comment:
Para que os pase este test:
it("may create objects also with the module pattern", function() {
      function createObject(initialScore, color) {
        this.score = initialScore
        this.color = color
        this.incrementScoreIn = function(value){
          this.score += value;
        }
        this.points = function(){return this.score}
      }

      
      var obj = new createObject(5, 'red');
      obj.incrementScoreIn(5);
      expect(obj.color).toEqual('red');
      expect(obj.points()).toEqual(10);
      
    });